rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -361,6 +361,8 @@ extern void pre_main (void);
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE / sizeof(uint32_t)];
 #elif defined(TARGET_XDOT_L151CC)
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE * 6 / sizeof(uint32_t)];
+#elif defined(TARGET_REALTEK_RTL8195AM)
```

Comment:
The CI test put its objects in main thread stack while our SPI implementation takes a large chunk of memory when GDMA is switch on. That means if we want 4K size main thread stack and passing CI test at the same time, we need to switch off GDMA for our SPI. @bulislaw does that acceptable?